src/components/InputForm.tsx

```diff
@@ -0,0 +1,85 @@
+import useInput from '../hooks/useInput';
+import styled from 'styled-components';
+import { IInputFormProps } from './Interface';
+
+const InputForm = ({
+  currentUserId,
+  chatList,
+  setChatList,
+}: IInputFormProps) => {
+  const { inputText, handleInputChange, reset } = useInput('');
+
+  const handleNewMsg = (e) => {
```

Comment:
```suggestion
  const handleNewMsg = (e: React.MouseEvent<HTMLButtonElement>) => {
```
파라미터에 implicit any 타입이 사용되고 있습니다. 타입을 명시하는것이 좋을것 같습니다.